lib/ferrum/browser.rb

```diff
@@ -109,6 +109,14 @@ def resize(**options)
       page.resize(**options)
     end
 
+    def get_position
+      page.get_position
+    end
+
+    def set_position(left, top)
+      page.set_position(left, top)
+    end
+
```

Comment:
it can be delegated to page here:
https://github.com/rubycdp/ferrum/pull/116/files#diff-5af714046589275108757b6cc194e7ae6e35bc6bcdb723ecfa48aad1cf7232f0R20
```suggestion
    delegate :get_position, :set_position, to: :page

```